src/styles/main.scss

```diff
@@ -1,3 +1,34 @@
+@import "reset";
+
+@import "variables";
+
+@import "mixins";
+
+@import "header";
+
+@import "nav";
+
+@import "card";
+
+@import "stars";
```

Comment:
mb remove spaces between imports?

Reply:
Okay, I thought that It is will be easier to read (: